shopping_mall/backend/app/services/faq_gap_analyzer.py

```
@@ -0,0 +1,235 @@
+
+    if not gap_logs:
+        return GapAnalysisResult(period_days=days, total_gap_questions=0, items=[])
+
+    # ── 3. normalize_query 기반 텍스트 클러스터링 ────────────────────────────
+    # 정규화된 텍스트가 같으면 같은 질문으로 간주
+    clusters: dict[str, GapCluster] = {}
+
+    for log in gap_logs:
+        try:
+            key = normalize_query(log.question)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)